mtags-shared/src/main/scala/scala/meta/internal/pc/ReferencesRequest.scala

```diff
@@ -12,7 +12,9 @@ import org.eclipse.lsp4j.jsonrpc.messages.{Either => JEither}
 case class PcReferencesRequest(
     file: VirtualFileParams,
     includeDefinition: Boolean,
-    offsetOrSymbol: JEither[Integer, String]
+    offsetOrSymbol: JEither[Integer, String],
+    override val alternativeSymbols: ju.List[String] =
+      ju.Collections.emptyList()
```

Comment:
Do we need a default value here?

Reply:
No, but it makes sense for when you want to create params for offset.